### DISCUSSION TITLE
When will apt-maven-plugin be updated?

### DISCUSSION THREAD

**Discussion by wang3develop** [querydsl/querydsl#3197] [Show and tell]
![图片](https://user-images.githubusercontent.com/44887680/157429675-d364d7f0-9c77-4638-80d2-94755f2dd724.png)
![图片](https://user-images.githubusercontent.com/44887680/157429843-99cdae33-b0f1-4fa6-aaee-24744d4560f6.png)

**Comment by wang3develop** [comment]
Maybe I should turn to JOOQ

**Reply by natros** [reply]
Why do you need the apt-maven-plugin, Isn't maven-compiler-plugin enough?
But if you really need it, why do you not update the guava dependency in your project?

**Comment by jwgmeligmeyling** [comment]
The `apt-maven-plugin` is not part of this repository. The `apt-maven-plugin` is archived and no longer developed.

 If you need a plugin, you can use the `maven-compiler-plugin`. However, we recommend to use no plugin at all and just use the appropriate classifier for the `querydsl-apt` dependency. This automatically registers the APT plugin with the compiler.

```xml
              <dependency>
                <groupId>com.querydsl</groupId>
                <artifactId>querydsl-apt</artifactId>
                <version>${querydsl.version}</version>
                <classifier>jpa</classifier>
                <scope>provided</scope>
              </dependency>
```
